.circleci/config.yml

```diff
@@ -0,0 +1,71 @@
+version: 2
+
+setup_ftl: &setup_ftl
+  - run:
+      name: Authorize gcloud and set config defaults
+      command: |
+        echo $GCLOUD_SERVICE_KEY | base64 -di > ${HOME}/gcloud-service-key.json
+        sudo gcloud auth activate-service-account --key-file=${HOME}/gcloud-service-key.json
+        sudo gcloud --quiet config set project ${GOOGLE_PROJECT_ID}
+
+jobs:
+  build_and_setup:
+    docker:
+      - image: circleci/android:api-28-alpha  # gcloud is baked into this image
+    working_directory: ~/project
+    environment:
+      JVM_OPTS: -Xmx3200m
```

Comment:
You really want to use my working battle-tested environmental variables:

https://github.com/chrisbanes/tivi/blob/master/.circleci/config.yml